src/main/java/org/opentripplanner/routing/api/request/refactor/preference/WheelchairPreferences.java

```diff
@@ -0,0 +1,55 @@
+package org.opentripplanner.routing.api.request.refactor.preference;
+
+import java.io.Serializable;
+import javax.annotation.Nonnull;
+import org.opentripplanner.routing.api.request.WheelchairAccessibilityRequest;
+
+public class WheelchairPreferences implements Cloneable, Serializable {
+
+  /**
+   * Whether the trip must be wheelchair-accessible and how strictly this should be interpreted.
+   */
+  @Nonnull
+  private WheelchairAccessibilityRequest accessibility = WheelchairAccessibilityRequest.DEFAULT;
+
+  // TODO: 2022-08-18 Is it new? never used
+  private double maxSlope = 0.0833333333333;
+
+  // TODO: 2022-08-25 this was in specification but we never use it
+  // TODO: 2022-08-25 do we need it?
+  private double slopeTooSteepCostFactor = 10.0;
```

Comment:
It should be used in `org.opentripplanner.routing.edgetype.StreetEdgeReluctanceCalculator#computeWheelchairReluctance`

Seems the value from `WheelchairAccessibilityRequest` is used. I think we should remove this class, and use that class wherever needed.